src/pages/using-spark/components/tooltips.mdx

```diff
@@ -0,0 +1,58 @@
+---
+  title: Tooltips
+---
+import ComponentPreview from '../../../components/ComponentPreview';
+import { SprkDivider } from '@sparkdesignsystem/spark-react';
+import tooltipExample from '../../../images/tooltips.png';
+
+# Tooltips
+
+The tooltips component appears as a popup design providing a short text description of information about a feature when the user hovers over the item.
+
+<ComponentPreview
+  componentName="tooltip--default-story"
+  minHeight="10rem"
+  hasHTML
+/>
+
+### Usage
+
+Use tooltips when the feature needs to be explained in further detail to walk users through a specific action. It helps to draw the users' attention to a feature without interrupting their workflow.
+
+### Guidelines
+
+- Appears when a user hovers over or clicks on the tooltips icon.
+- Remains in view until the user actively dismisses.
+- Located above other content on the page, but below a modal or masthead.
+- Information should be discrete, short and concise.
+- Do not use as a warning or alert.
+- Do not include in links or modals.
+
+<SprkDivider
+ element="span"
+ additionalClasses="sprk-u-Measure"
+></SprkDivider>
+
+## Anatomy
+
+- Content within tooltips must be text based with a max character count of 140.
+- The maximum width of tooltips must be 290 px.
+- Position of design changes based on how close the feature is to the screen’s edge such as:
+
+<img
+  class="sprk-u-mbm"
+  src={tooltipExample}
+  alt="The Spark Tooltip component depicted in four different orientations."/>
+
+## Accessibility
+Tooltips must be discoverable and [operable](https://www.w3.org/TR/WCAG20/#operable) by users that rely on a diverse set of established navigational means. Also, in compliance with [1.4.13: Content on Hover or Focus](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html) WCAG rules, make sure that any content appearing on hover or focus is dismissible, hoverable, and persistent.
+
+- Must be accessible by mouse, pointer devices, keyboard-only, screen reader, zoom software, and any other assistive technology.
+- Allow the mouse to easily move over the tooltip without dismissing it.
```

Comment:
Are these items under the a11y section here additional work that people using our tooltip will do? That's how it is reading to me.